src/components/panes/CameraTest/CameraTest.tsx

```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles((theme: Theme) =>
       padding: '2em',
       borderRadius: '8px',
       maxWidth: '450px',
-      [theme.breakpoints.down('sm')]: {
+      [theme.breakpoints.down('md')]: {
```

Comment:
@timmydoza I made this change because I noticed the camera test looked comically small on my iPad Air:
![IMG_0008](https://user-images.githubusercontent.com/77076398/154178628-4717e29d-bd41-4dd0-9114-f51d77844ab6.PNG)

here is what it looks like when we use the `'md'` breakpoint:

![IMG_0009](https://user-images.githubusercontent.com/77076398/154178736-07f9127a-c2d7-47dd-a724-c55fd3ba1c49.PNG)

I made the same change for the `<AudioTest />`